pkg/loadbalancer/proxy.go

```diff
@@ -50,14 +51,37 @@ static_resources:
   {{- range $index, $data := .ServicePorts }}
   - name: listener_{{$index}}
     address:
-      socket_address: { address: {{ $data.Listener.Address }}, port_value: {{ $data.Listener.Port }} }
+      socket_address:
+        address: {{ $data.Listener.Address }}
+        port_value: {{ $data.Listener.Port }}
+        protocol: {{ $data.Listener.Protocol }}
+    {{- if eq $data.Listener.Protocol "UDP"}}
+    udp_listener_config:
+      downstream_socket_config:
+        max_rx_datagram_size: 9000
```

Comment:
again, not clear why this needs to be specified?

Reply:
copy paste from the totally obscure envou config world

Reply:
https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/udp_filters/udp_proxy#example-configuration